atl-checker/src/algorithms/certain_zero/search_strategy/mod.rs

```diff
@@ -24,6 +24,12 @@ pub trait SearchStrategy<V: Vertex> {
     fn queue_back_propagation(&mut self, edge: Edge<V>) {
         self.queue_new_edges(vec![edge])
     }
+
+    /// Modify the edges to benefit the search strategy. For instance, this could be a
+    /// rearrangement of the targets of the edges which results in better performance.
+    fn modify(&mut self, edge: Vec<Edge<V>>) -> Vec<Edge<V>> {
+        return edge;
```

Comment:
```suggestion
        edge
```
This should fix the warning